addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -84,7 +86,12 @@
     private static final Map<String, Pattern> WIN_OS_PAYLOADS = new LinkedHashMap<>();
     private static final Map<String, Pattern> PS_PAYLOADS = new LinkedHashMap<>();
 
+    private static final Map<String, String> ALERT_TAGS = new HashMap<>();
+
     static {
+        ALERT_TAGS.put(AlertTags.OWASP_2021_A03_KEY, AlertTags.OWASP_2021_A03_VALUE);
+        ALERT_TAGS.put(AlertTags.OWASP_2017_A01_KEY, AlertTags.OWASP_2017_A01_VALUE);
+
```

Comment:
Should this be a separate static block from the payloads?

Reply:
+1